ballerina/types.bal

```diff
@@ -23,3 +23,48 @@ public type FieldMetadata record {|
 |};
 
 public type FilterQuery distinct sql:ParameterizedQuery;
+
+# Defines the entities of the table.
+#
+# + key - The primary key of the table
+# + unique - The unique index of the tavle
+# + tableName - The table name
+public type EntityConfig record {|
+    string[] key;
+    string[][] unique?;
+    string tableName?;
+|};
+
+# The annotation is used to represent entities such as primary keys, unique indexes, and table name.
```

Comment:
Should this be `The annotation is used to represent attributes of an entity such as primary keys, unique indexes, and table name.` instead?